api/GPUQueue.json

```
@@ -344,6 +344,47 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "unique_command_buffers": {
```

In https://github.com/mdn/browser-compat-data/pull/24811 we used the `validates_x` pattern for the feature key, so how about:

```suggestion
        "validates_command_buffer_uniqueness": {
```

---

Ah, good call. Committing.